Prototope/Layer+Layout.swift

```diff
@@ -66,6 +66,30 @@ extension Layer {
 	}
 	
 	
+	/** Moves the receiver so that its left side is aligned with the left side of its parent layer. */
+	public func moveToLeftSideOfParentLayer(margin: Double = 0.0) {
+		if let parent = self.parent {
+			self.frameMinX = -margin
```

Comment:
I think this is backwards still. This method tells me “Move my originX (frameMinX) to the leftmost side of my parent (i.e., x = 0) but then offset it with a margin towards the right.

```
layer.moveToLeftSideOfParentLayer() // aligns the layer flush against its parent’s originX
layer.moveToLeftSideOfParentLayer(margin: 10) // essentially sets the layer’s originX to be 10
```